feature/sessions/src/main/java/io/github/droidkaigi/confsched2022/feature/sessions/Search.kt

```diff
@@ -281,52 +285,72 @@ private fun SearchedItemListField(
                 SearchedHeader(day = dayToTimeTable)
             }
             items(sessions) { timetableItemWithFavorite ->
-                Box(
+                val actionLabel = stringResource(
+                    if (timetableItemWithFavorite.isFavorited) {
+                        Strings.unregister_favorite_action_label
+                    } else {
+                        Strings.register_favorite_action_label
+                    }
+                )
+                Row(
                     modifier = Modifier
                         .fillMaxWidth()
                         .clickable { onItemClick(timetableItemWithFavorite.timetableItem.id) }
                         .padding(16.dp)
+                        .semantics(mergeDescendants = true) {
+                            customActions = listOf(
+                                CustomAccessibilityAction(
```

Comment:
https://user-images.githubusercontent.com/13705006/191950425-9b7457a0-db53-4e77-a431-b25deef4b2bf.mp4

Favorite operation using Talkback has been changed to CustomAction.


ref: https://youtu.be/6LsaP6oKxMY?t=764